load_testing/justfile

```diff
@@ -6,7 +6,10 @@ build:
     docker build . -t openverse-load-testing:latest
 
 run script host: build
-    docker run --rm -it -v {{invocation_directory()}}:/app openverse-load-testing:latest ./{{ script }}.sh {{ host }}
+    docker run --rm -it -v {{ invocation_directory() }}:/app openverse-load-testing:latest ./{{ script }}.sh {{ host }}
 
 api host=("http://" + DOCKER_HOST + ":8000"):
     @just run api {{ host }}
+
+k6 env="-e API_URL=https://api-dev.openverse.engineering/v1/": build
```

Comment:
Given we also have to pass in the `API_TOKEN` (otherwise the page_size=500 requests will fail), it could be nice for environment variables to be explicitly given as inputs rather than having to pass them individually through `-e` docker run syntax.

What do you think? Or maybe we should encourage the use of a .env file for the justfile? Just loads .env's automatically from what I understand as long as we update `set dotenv-load := false` to `set dotenv-load := true`.

Reply:
I like the use of a .env file and made those changes, that is what you were thinking with "enforcing" it?

Reply:
Yes! That works great. Would we want to pick up the API_URL from there as well? :thinking: